046_numberAsSumOfprimenumber.js

```diff
@@ -0,0 +1,30 @@
+// 46. Can a number be expressed as a sum of two prime numbers?
+const primeNumber = (number) => {
+  let primeNumber = true;
```

Comment:
Function name "primeNumber" and the variable name "primeNumber" should not be similar. It works, but by convention, it is should be written differently